js/templates/modals/editListing/variantInventory.html

```diff
@@ -3,15 +3,15 @@
 <% } else { %>
 <div class="inventoryTableWrap">
   <table>
-    <tr>
```

Comment:
A couple things:

1.) The sizing seems a bit off in that the variant table is unnecessarily getting a horizontal scrollbar it wasn't getting before. There are certain situations where it's unavoidable (e.g. if you're tracking inventory since that adds in an extra checkbox), but by default there wasn't one there before.

2.) The header row has a different border than it used to have - I'm not sure that's necessary. Also, the header labels used to be bold and they no longer are.

Before:

![image](https://user-images.githubusercontent.com/794517/34389871-f594112c-eaf0-11e7-8fa8-9cce69ce195d.png)

After:

![image](https://user-images.githubusercontent.com/794517/34389875-023d84d0-eaf1-11e7-8d40-ea5647e7800a.png)